tests/helpers.py

```diff
@@ -387,14 +391,23 @@ def delete_cephblockpool():
     return True
 
 
-def create_cephfilesystem():
+def create_cephfilesystem(force_create=True):
     """
-    Function for deploying CephFileSystem (MDS)
+    Function for deploying CephFileSystem (MDS).
+
+    Args:
+        force_create (bool): If true it will delete old CFS and recreate new one, If False will skip creation of CFS
 
     Returns:
         bool: True if CephFileSystem creates successful
     """
     fs_data = copy.deepcopy(defaults.CEPHFILESYSTEM_DICT)
+    if force_create:
+        logger.info("Deleting CephFileSystem if exists")
+        delete_all_cephfilesystem()
+    else:
```

Comment:
this still doesn't answer the situation in which you have a leftover FS and you want to know about it. I think it would be better to assert in case a FS already exists